docs/design/architecture.md

```diff
@@ -2,128 +2,353 @@
 
 ## Overview
 
-This is an architectural overview of Kata Containers, based on the 2.0 release.
+This document is a summary of the Kata Containers architecture.
 
-The primary deliverable of the Kata Containers project is a CRI friendly shim. There is also a CRI friendly library API behind them.
+Kata Containers provides a container runtime that creates containers
+inside lightweight, hardware isolated and secure virtual machines
```

Comment:
security is a spectrum.  I would rather see something like what's on katacontainers.io:
```
Kata Containers is an open source community working to build a secure container runtime with lightweight virtual machines that feel and perform like containers, but provide stronger workload isolation using hardware virtualization technology as a second layer of defense.
```

Reply:
@egernst - agreed. Updated with light modifications to your suggested wording.